source/includes/administration/_invoices.md

```diff
@@ -2136,3 +2136,52 @@ curl  -X GET 'https://cloudmc_endpoint/rest/invoices/download?invoice_id=3f7b7cc
 | Required | &nbsp; |
 | --- | --- |
 | `invoice_id`<br/>*UUID* | The id of the invoice. |
+
+
+<!-------------------- FLAG INVOICE -------------------->
+
+`POST /invoices/:invoiceId/flag`
+
+Flag an invoice with a message to prevent the invoice from being automatically issued to the customer. 
+Once an invoice is flagged, pricing configuration changes can be made. 
```

Comment:
the wording implies changes can't be made unless it's flagged imo

Reply:
[`c9b3740` (#501)](https://github.com/cloudops/cloudmc-api-docs/pull/501/commits/c9b3740770f3467a99dcb14babb00eb5cc8eda90)